sema/ro/creator/rocreator.py

```
@@ -0,0 +1,56 @@
+    @property
+    def success(self) -> bool:
+        return self._success
+
+
+class Roc(ServiceBase):
+    """The main class for the roc 'ro-creator' service."""
+
+    def __init__(
+        self,
+        *,
+        root: str,
```

Ruff (0.8.2)

Unused method argument: `root`

(ARG002)